src/patternfly/demos/Modal/examples/modal-scroll-example.hbs

```diff
@@ -1,7 +1,7 @@
 {{#> modal}}
   {{#> backdrop}}
     {{#> bullseye}}
-      {{#> modal-box modal-box--attribute='aria-labelledby="modal-scroll-title" aria-describedby="modal-scroll-description"'}}
+      {{#> modal-box modal-box--modifier="pf-m-sm" modal-box--attribute='aria-labelledby="modal-scroll-title" aria-describedby="modal-scroll-description"'}}
         {{#> modal-box-close}}
           {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Close Dialog"'}}
             <i class="fas fa-times"></i>
```

Comment:
doesn't relate to this issue, but would be good to swap that to `{{> button-icon button-icon--type="times"}}`

Reply:
nice! updated.